src/IO/Observer/RegisterObservers.hpp

```diff
@@ -8,19 +8,19 @@
 #include "DataStructures/DataBox/DataBox.hpp"
 #include "IO/Observer/ObservationId.hpp"
 #include "IO/Observer/TypeOfObservation.hpp"
-#include "Time/Tags.hpp"
 
 namespace observers {
 /// Passed to `RegisterWithObservers` action to register observer event.
-template <typename ObsType>
+template <typename TemporalId, typename ObsType>
```

Comment:
I think we are consistent in calling `Tags::TimeStepId` the temporal ID in the evolution code, not `Tags::Time`, which is what is passed here.  Can you rename these parameters and the `temporal_id` variables?

Reply:
Good point. Renamed to `ObservationIdTag`. Please check the fixup.